src/mcp_server_helm/commands/basic.py

```
@@ -0,0 +1,64 @@
+import logging
+from typing import Optional
+from ..core.utils import execute_helm_command
+
+logger = logging.getLogger(__name__)
+
+
+def helm_completion(shell: str) -> str:
+    """
+    Generates the autocompletion script for the specified shell.
+    """
+    logger.info(f"Running helm completion with shell={shell}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)